tensor_genn/algorithms/relu_ann.py

```diff
@@ -242,31 +246,62 @@ def convert(self, tf_model, scaled_tf_weights=None):
 
         return self.g_model, self.neuron_pops, self.current_source
 
-    def evaluate(self, X, y=None):
+    def evaluate(self, X, y=None, raster_plot=False):
```

Comment:
I don't really like having the raster plotting functionality **in** this function - for a start it'll need to be duplicated in every other algorithm and also it doesn't allow users to do anything else with the data. Perhaps a better API would be:
```python
def  evaluate(self, X, y=None, save_example_spikes=[]):
```